CONTRIBUTING.md

```diff
@@ -87,6 +87,8 @@ npm run start:api --project_path=<path-to-Kedro-project>
 
 This command will launch a Kedro-Viz server at [localhost:4142](http://localhost:4142) and serve data from a real Kedro pipeline located at the project path supplied to the command.
 
+From then on, launching the app locally at [localhost:4141] will pull data from the Kedro-Viz server that is running on port 4142. (note: once the development server is launched at port 4142 the local app will always pull data from that server - to disable that, you could comment out the proxy setting in `package.json` and relaunch your app.)
```

Comment:
Wording/formatting suggestion:

```suggestion
This command will launch a Kedro-Viz server at [localhost:4142](http://localhost:4142) and serve data from a real Kedro pipeline located at the project path supplied to the command. From then on, launching the app locally at [localhost:4141](http://localhost:4141) will pull data from the Kedro-Viz server that is running on port 4142.

Note: Once the development server is launched at port 4142, the local app will always pull data from that server. To prevent this, you can comment out the proxy setting in `package.json` and relaunch your app.
```

Reply:
> relaunch your app

By this, do you mean restart the dev server at 4141? If so, I'd say that instead.

Reply:
sounds good, I'll state that instead.